src/main/java/mafia/mafiatogether/service/dto/RoomInfoResponse.java

```diff
@@ -32,19 +33,24 @@ public static RoomInfoResponse of(
         );
     }
 
-    private static List<PlayerResponse> convertFrom(final Player player, final Map<String, Player> players) {
-        if (!player.isAlive()) {
-            return players.values().stream()
-                    .map(PlayerResponse::forDead)
-                    .toList();
+    private static List<PlayerResponse> convertFrom(final Player owner, final Map<String, Player> players) {
+        PlayerResponse myJob = PlayerResponse.forMyJob(players.get(owner.getName()));
+        List<PlayerResponse> responses = players.values().stream()
+                .filter(it -> !it.getName().equals(owner.getName()))
+                .map(it -> playerToResponse(owner, it))
```

Comment:
```suggestion
                .filter(response -> !response.getName().equals(owner.getName()))
                .map(response -> playerToResponse(owner, response))
```
it보다는 response 하나식을 의미하는거니까 이렇게 하는건 어때?